server/index.js

```diff
@@ -1,9 +1,27 @@
 const express = require('express');
-
+const bodyParser = require('body-parser');
+const PriceVolume = require('../database/PriceVolume.js');
 const app = express();
 const PORT = 3002;
 
 app.use(express.static(__dirname+'/../client/dist'));
+// parse application/x-www-form-urlencoded
+app.use(bodyParser.urlencoded({ extended: false }))
+// parse application/json
+app.use(bodyParser.json())
+
+
+app.get('/api/volumes/symbols/:symbolId', function(req, res){
+  console.log(req.params.symbolId);
```

Comment:
Once you've confirmed that this is what you expect, you can take it out

Reply:
Yes. I will take out all console.logs.